utils/eventListeners.ts

```diff
@@ -136,7 +136,15 @@ export const waitForEvent = async (
   api: ApiPromise,
   method: string,
   blocks: number = 10
-): Promise<void> => {
+): Promise<any[]> => {
+  return (await waitForEvents(api, method, blocks))[0]
+}
+
+export const waitForEvents = async (
+  api: ApiPromise,
+  method: string,
+  blocks: number = 10
+): Promise<any[][]> => {
```

Comment:
Optional: let's try to avoid "any" return type plz!

Reply:
object would be better ?